Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,79 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
+This is the default end-of-checkout script.
+NOTE: Prepare to take fltas only if you have sufficient time to vent the telescope afterwards.
```

Comment:
fltas = flats

Reply:
Also, if you want to make this a new paragraph, then you need to add a blank line before it.
However, rst documentation has a special functionality exactly for this, which renders it in a nice special block 

See this for info (but isn't really the official help page):
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes

See this link for the more official help (which is more complete but less user friendly):
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-note

`
.. note::
   Only run the prepare to take flats script if you have sufficient time to vent the telescope afterwards.

`